code/datums/achievements/misc_achievements.dm

```diff
@@ -141,3 +141,9 @@
 	name = "Little Chickadee"
 	desc = "No, well, ten beers"
 	database_id = MEDAL_DRUNK
+
+/datum/award/achievement/misc/cayenne_disk
```

Comment:
No beecoin reward?

Reply:
The other achievements don't have a reward set too.

Reply:
Scroll up further.

Reply:
🙄